### ISSUE TITLE
Level: "Meander"

### ISSUE THREAD

**Issue by tttppp** [andybalaam/rabbit-escape#74]
A smaller level - maybe more suitable to mobile screens.
```
:name=Meander
:description=Just go with the flow
:num_rabbits=5
:num_to_save=5
:bridge=5
:dig=2
:climb=1
:rabbit_delay=5
:music=tryad-witness
###############
#Q#     #     #
# # ### # ### #
# # #   # #   #
# # # ### # ###
#   #     #  O#
###############
```

**Comment by colonelfazackerley** [comment]
Took me 15 min or so. Not sure I did it the right way. My solution also relies on something that might not be intended in the game design. I can post a pic, but I will wait and see if Andy wants to figure it out for himself.

Hope he gets some internet back soon. the content is piling up here. We'll need 40 extra levels in the next release.

**Comment by tttppp** [comment]
If the weird thing you used was stacking bridges then that's what I had in mind - see #34 where Andy says "I used 1 "weird" but deliberate behaviour of stacking bridges..."

**Comment by colonelfazackerley** [comment]
It was a bridge stack.

**Comment by andybalaam** [comment]
Stupid stupid stupid Internets

**Comment by colonelfazackerley** [comment]
merged to master f96dd5696bc0d3c0f44b4856ba75e3a7d2bc201a
close me

**Comment by andybalaam** [comment]
I just solved this using the weird walk on air after climbing bug. Looking at the discussion above there must be a better way.

**Comment by andybalaam** [comment]
Hint 2 has an extra backslash

**Comment by andybalaam** [comment]
Yes I did this using bridge stacking and I consider that solution legit.

**Comment by andybalaam** [comment]
Hints fixed in 520621268bc72e5f717eb62aaf3c0d5e87138721
